nautobot/extras/api/views.py

```diff
@@ -288,6 +289,11 @@ def run(self, request, class_path):
         if commit is None:
             commit = getattr(job_class.Meta, "commit_default", True)
 
+        try:
+            job.validate_data(data)
+        except ValidationError as e:
+            return Response(e.message, status=400)
```

Comment:
Can we add some test coverage for this in `nautobot/extras/tests/test_api.py` (`JobTest`)?

Reply:
Added test cases for all failure scenarios (malformed data, missing fields, unknown fields).